src/Assets/Styles/site.scss

```
@@ -206,3 +205,8 @@ $govuk-global-styles: true;
     display: none;
   }
 }
+
+// Autocomplete overrides
+.autocomplete__wrapper {
+  font-family: "nta", Arial, sans-serif;
```

Very minor but we can use the variable here
```suggestion
  font-family: $font-family;
```